.github/workflows/build_latest_image.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
```

Comment:
Nitpick: Isn't just the last step to push it? I at least includes the build as well. 🤷🏾

Reply:
Tried to improve it.

Reply:
Very good. 👍🏾